docs/src/further_topics/filtering_warnings.rst

```diff
@@ -128,7 +128,9 @@ Or you can target Warnings raised by specific lines of specific modules, e.g.
     ...     warnings.filterwarnings("ignore", module="iris.coord_systems", lineno=442)
     ...     my_operation()
     ...
-    iris/coord_systems.py:768: IrisDefaultingWarning: Discarding false_easting and false_northing that are not used by Cartopy.
+    iris/coord_systems.py:444: IrisUserWarning: Setting inverse_flattening does not affect other properties of the GeogCS object. To change other properties set them explicitly or create a new GeogCS instance.
+      warnings.warn(wmsg, category=iris.warnings.IrisUserWarning)
+    iris/coord_systems.py:770: IrisDefaultingWarning: Discarding false_easting and false_northing that are not used by Cartopy.
```

Comment:
```suggestion
      warnings.warn(wmsg, category=iris.warnings.IrisUserWarning)
    iris/coord_systems.py:770: IrisDefaultingWarning: Discarding false_easting and false_northing that are not used by Cartopy.
```

This doctest is supposed to demonstrate the filtering of a warning based on line number. The `IrisUserWarning` is now being generated on line `444`, so a filter with `lineno=442` is not a valid demonstration. The correct fix would be to change to `lineno=444`.

Reply:
Still need to change:

```diff
-    ... warnings.filterwarnings("ignore", module="iris.coord_systems", lineno=442)
+    ... warnings.filterwarnings("ignore", module="iris.coord_systems", lineno=444)
```

Sorry if that wasn't clear in my previous comment.